hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClientThrottlingIntercept.java

```diff
@@ -124,23 +125,24 @@ static AbfsClientThrottlingIntercept initializeSingleton(AbfsConfiguration abfsC
    * @return true if the operation is throttled and has some bytes to transfer.
    */
   private boolean updateBytesTransferred(boolean isThrottledOperation,
-      HttpOperation abfsHttpOperation) {
+      AbfsHttpOperation abfsHttpOperation) {
     return isThrottledOperation && abfsHttpOperation.getExpectedBytesToBeSent() > 0;
   }
 
   /**
    * Updates the metrics for successful and failed read and write operations.
+   *
    * @param operationType Only applicable for read and write operations.
-   * @param abfsHttpOperation Used for status code and data transferred.
+   * @param httpOperation Used for status code and data transferred.
```

Comment:
nit : param name can be kept abfsHttpOperation only

Reply:
taken.